source/main/index.js

```diff
@@ -104,6 +109,10 @@ const onAppReady = async () => {
     environment.version,
     path.join(pubLogsFolderPath, 'Daedalus-versions.json')
   );
+  // TODO: we should only use this store instance on main process
+  // There are several other places that store instances are created
+  // unnecessarily (it doesn't appear to be a big issue though)
+  const store = new Store();
```

Comment:
What about `source/main/ipc/electronStoreConversation.js`? That file seems to have some structure to be consumed by the other ones, like `source/main/utils/mainLocalStorage.js`

Reply:
Good point, I haven't seen those implementations yet 👍 will refactor this PR to use these